privacyidea/static/components/config/views/config.resolvers.html

```diff
@@ -25,8 +25,13 @@
                     $state.includes('config.resolvers.add'+resolvertype) }">
                     <a ui-sref="config.resolvers.add{{ resolvertype }}"
                             translate>
-                        New {{ resolvertype }}</a>
-                </li>
+                        New {{ resolvertype }}</a></li>
+                <hr>
+                <li role="presentation">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/configuration/useridresolvers.html#'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about UserIdResolvers</span></a></li>
```

Comment:
I think "UserResolver" would be a better word, even though the documentation says userIdResolver...

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.